test/PowerTools.Test/packages.config

```diff
@@ -1,5 +1,5 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
-  <package id="Moq" version="4.0.10827" />
+  <package id="Moq" version="4.0.10827" requireReinstallation="True" />
```

Comment:
Whenever you change target framework, NuGet flags them for reinstall, but this version of Moq only has a net40 library, so it's not required.